### ISSUE TITLE
Refactor method Aether.resolve(...)

### ISSUE THREAD

**Issue by amihaiemil** [jcabi/jcabi-aether#86]
In method ``public List<Artifact> resolve(final Artifact root,
        final String scope, final DependencyFilter filter)``, variables ``rdep`` and ``crq`` are both used only once and can be refactored from: 
```
final Dependency rdep = new Dependency(root, scope);
final CollectRequest crq = this.request(rdep);
```
to this method call:
``this.request(new Dependency(root, scope))``

**Comment by dmarkov** [comment]
@yegor256 please take a look at it and dispatch (see [par.21](http://at.teamed.io/policy.html#21))

**Comment by amihaiemil** [comment]
@yegor256 can you have a look here pls? Thanks

**Comment by amihaiemil** [comment]
@yegor256 ping

**Comment by amihaiemil** [comment]
@yegor256 Don't forget about this pls

**Comment by yegor256** [comment]
@amihaiemil fixed in 5a2c37a thanks
